src/application/components/Cache/Cache.tsx

```diff
@@ -78,15 +51,21 @@ export function Cache({
             />
           </Fragment>
         ) : (
-          <h3 css={noDataStyles}>No cache data</h3>
+          <h3 className="ml-3 uppercase text-sm font-normal pt-4 text-white/50 tracking-wider">
```

Comment:
On the flipside, as more styles are needed, this can grow to be a bit large, which is the most commonly stated downside of Tailwind 😆